app.yaml.base

```diff
@@ -38,15 +38,10 @@ handlers:
           static_files: static/\1
           upload: static/(.*\.(bmp|gif|ico|jpeg|jpg|png))
 
-# site root
-        - url: /
-          static_files: static/index.html
-          upload: static/index.html
-          expiration: "15m"
-
 # All URLs for Back End logic
-        - url: /api/.*
+        - url: /.*
```

Comment:
This is wrong.

- lines 48+ now are ignored
- you skip static handlers entirely